src/views/Simulation/components/InspectorDrawer/InspectorDrawer.js

```diff
@@ -23,7 +32,35 @@ const InspectorDrawer = ({ selectedElement, setSelectedElement }) => {
   const { width, startResizing } = useResizableDrawer();
 
   const inspectedElement = useMemo(() => selectedSubElement ?? selectedElement, [selectedElement, selectedSubElement]);
-  const isResource = useMemo(() => inspectedElement?.operationsCount != null, [inspectedElement]);
+  const isResource = useMemo(() => inspectedElement?.type === 'productionResource', [inspectedElement]);
+
+  const demandChart = useMemo(() => {
+    const data = graphRef.current?.stockDemands?.[inspectedElement?.id];
+    if (!data || inspectedElement?.type !== 'stock') return null;
+    return (
+      <>
+        <Typography variant="h6" fontWeight="fontWeightBold">
+          Demand
+        </Typography>
+        <InspectorChart data={data} width={width - CHART_WIDTH_OFFSET} height={CHART_HEIGHT} chartColor="#40E0D0" />
```

Comment:
Maybe later but I think we should set all colors palettes in a separated const file.